### DISCUSSION TITLE
Custom notification text

### DISCUSSION THREAD

**Discussion by ririko5834** [upptime/upptime#274] [Ideas]
Hello, could you please add that you can specify custom notification message in .env?
It should be NOTIFICATION_TEXT variable and there should be variables that you can use:
- {EMOJI} - status emoji (🟩, 🟨 or 🟥)
- {SITE_NAME} - Site name
- {SITE_URL}  - Site url
- {SITE_STATUS} - Site status (is down, has degraded performance and is back up)
- {ISSUE_URL} - Url to github issue (example: https://github.com/upptime/upptime/issues/269) 
- {INCIDENT_URL} - Url to incident on status page (example: https://demo.upptime.js.org/incident/269)

I think that option to setup custom notification text will users like 👍

**Comment by wolfier74** [comment]
Totally agree with that!

**Comment by K1ngJony** [comment]
Created a PR for this (https://github.com/upptime/upptime.js.org/pull/44). Although, looking through the code, INCIDENT_URL isn't a current variable that can be used.
